pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -413,6 +413,18 @@
     )
     private int brokerDeduplicationMaxNumberOfProducers = 10000;
 
+    @FieldContext(
+        category = CATEGORY_POLICIES,
+        doc = "How often is the thread pool scheduled to check whether a snapshot needs to be taken.(disable with value 0)"
+    )
+    private int brokerDeduplicationSnapshotFrequencyInSeconds = 120;
+    @FieldContext(
+        category = CATEGORY_POLICIES,
+        doc = "If this time interval is exceeded, a snapshot will be taken."
+            + "It will run simultaneously with `brokerDeduplicationEntriesInterval`"
+    )
+    private Integer brokerDeduplicationSnapshotIntervalSeconds = 120;
```

Comment:
Why Integer and not int?

Reply:
Later, we need to support the namespace-level and topic-level. When it is null, it means that this level is not set. Therefore the packaging type is used.

Reply:
Makes sense.
Okay.
Thanks for your clarification